<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grupoOpcoes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>651</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>431</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>       Menu de Opções</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameOpcoes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>181</width>
       <height>391</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="labelMovimentoDaJanela">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Window Movement</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>6</number>
      </property>
     </widget>
     <widget class="QFrame" name="frameMovimentoDaJanela">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>141</width>
        <height>281</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="buttonZoomOut">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonZoomIn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonDown">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>DOWN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRight">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonLeft">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>LEFT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonUp">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>UP</string>
       </property>
      </widget>
      <widget class="QSlider" name="zoomSlider">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>93</red>
             <green>156</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>61</red>
             <green>87</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>93</red>
             <green>156</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>61</red>
             <green>87</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>215</red>
             <green>215</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>61</red>
             <green>87</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="sliderPosition">
        <number>10</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextBrowser" name="zoomText">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QLabel" name="labelPercent">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>16</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Resetar Posição</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="labelObjetos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Objetos</string>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
    <widget class="QListWidget" name="listaObjetos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Novo Objeto</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="botaoRemoverObjeto">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover Objeto</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="logText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>661</width>
      <height>91</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>209</red>
          <green>209</green>
          <blue>209</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>209</red>
          <green>209</green>
          <blue>209</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>660</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelViewport">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Viewport</string>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>530</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCG1">
    <property name="title">
     <string>CG1</string>
    </property>
    <addaction name="tabNovo"/>
   </widget>
   <widget class="QMenu" name="menuInserir">
    <property name="title">
     <string>Inserir</string>
    </property>
    <addaction name="tabNovoObjeto"/>
   </widget>
   <addaction name="menuCG1"/>
   <addaction name="menuInserir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo_Objeto">
   <property name="text">
    <string>Novo Objeto</string>
   </property>
  </action>
  <action name="tabNovoObjeto">
   <property name="text">
    <string>Novo Objeto</string>
   </property>
  </action>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="tabNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listaObjetos</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>abrirJanela()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabNovoObjeto</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>abrirJanela()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moverParaCima()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDown</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moverParaBaixo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRight</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moverParaDireita()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonLeft</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>moverParaEsquerda()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonZoomIn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonZoomOut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoRemoverObjeto</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>destruirFigura()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabNovo</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>reiniciar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>mostrarValorDoZoom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetarWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abrirJanela()</slot>
  <slot>moverParaCima()</slot>
  <slot>moverParaBaixo()</slot>
  <slot>moverParaEsquerda()</slot>
  <slot>moverParaDireita()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>destruirFigura()</slot>
  <slot>reiniciar()</slot>
  <slot>mostrarValorDoZoom(int)</slot>
  <slot>resetarWindow()</slot>
 </slots>
</ui>
